### PR TITLE
add rate limiter config

### PR DESCRIPTION
**Pull request by shiyan2016**
### Ⅰ. Describe what this PR does
add rate limiter config

### CODE REVIEW

File: pkg/controller/cloneset/cloneset_controller.go [hunk 1]
```diff
@@ -99,7 +101,9 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 // add adds a new Controller to mgr with r as the reconcile.Reconciler
 func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Create a new controller
-	c, err := controller.New("cloneset-controller", mgr, controller.Options{Reconciler: r, MaxConcurrentReconciles: concurrentReconciles})
+	c, err := controller.New("cloneset-controller", mgr, controller.Options{
+		Reconciler: r, MaxConcurrentReconciles: concurrentReconciles,
+		RateLimiter: workqueue.NewItemExponentialFailureRateLimiter(util.BaseDelay, util.MaxDelay)})
```
Comment (FillZpp) [inline pkg/controller/cloneset/cloneset_controller.go:106]: The default rate limiter is https://github.com/openkruise/kruise/blob/master/vendor/k8s.io/client-go/util/workqueue/default_rate_limiters.go#L39 , including `ItemExponentialFailureRateLimiter` and `BucketRateLimiter`.
We should not just use `ItemExponentialFailureRateLimiter` here.

File: pkg/util/rate_limiter.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package util
+
+import (
+	"flag"
+	"time"
+)
+
+func init() {
+	flag.DurationVar(&BaseDelay, "rate-limiter-base-delay", time.Millisecond*5, "The base delay for rate limiter. Defaults 5ms")
+	flag.DurationVar(&MaxDelay, "rate-limiter-max-delay", time.Minute, "The max delay for rate limiter. Defaults 1min")
```
Comment (FillZpp) [inline pkg/util/rate_limiter.go:10]: `time.Minute` might break the default `1000s`

**Review by FillZpp — CHANGES_REQUESTED** [review]
The default rate limiter is https://github.com/openkruise/kruise/blob/master/vendor/k8s.io/client-go/util/workqueue/default_rate_limiters.go#L39 , including `ItemExponentialFailureRateLimiter` and `BucketRateLimiter`.
We should not just use `ItemExponentialFailureRateLimiter` here.

File: pkg/controller/cloneset/cloneset_controller.go (hunk 1 above)
Comment (shiyan2016) [reply pkg/controller/cloneset/cloneset_controller.go]: done

File: pkg/util/rate_limiter.go (hunk 2 above)
Comment (shiyan2016) [reply pkg/util/rate_limiter.go]: done

**Review by FillZpp — APPROVED** [review]
LGTM

**Pull request merged**